.gitignore

```diff
@@ -13,6 +13,8 @@ local.properties
 .settings/
 .loadpath
 .recommenders
+kami_Client.launch
+kami_Server.launch
```

Comment:
What are these referring to?

Reply:
I undid the change, they're eclipses launch files and you already added them to gitignore